src/FSLibrary/MissionMaxTPSMixed.fs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionMaxTPSMixed
+
+// This module provides a max TPS test with a blend of classic and Soroban load.
+// It uses the MaxTPSTest module to perform a binary search for the max TPS.
+
+open MaxTPSTest
+open StellarMissionContext
+open StellarCoreHTTP
+
+// If `list` is empty, return `value`. Otherwise, return `list`.
+let private defaultListValue value list =
+    match list with
+    | [] -> value
+    | _ -> list
+
+let maxTPSMixed (baseContext: MissionContext) =
+    let context =
+        { baseContext with
+              coreResources = SimulatePubnetTier1PerfResources
+              installNetworkDelay = Some(baseContext.installNetworkDelay |> Option.defaultValue true)
+              // Simulate apply duration using same distribution as
+              // `MissionSimulatePubnet`
+              simulateApplyDuration = Some(baseContext.simulateApplyDuration |> Option.defaultValue (seq { 0 }))
+              simulateApplyWeight = Some(baseContext.simulateApplyWeight |> Option.defaultValue (seq { 100 }))
+              enableTailLogging = false
+              // Setup distributions based on testnet data
+              wasmBytesDistribution =
+                  defaultListValue
+                      [ (8 * 1024, 132); (24 * 1024, 68); (40 * 1024, 92); (56 * 1024, 141) ]
```

Comment:
is it intended that the weights are not normalized? normalizing would help with readability